src/components/confirm/confirm.stories.mdx

```diff
@@ -74,6 +74,171 @@ const MyComponent = ({ isOpen, handleClose, handleConfirm }) => (
   </Story>
 </Preview>
 
+
+### Single action
```

Comment:
I think you need to add another hash tag to all the titles after this `Single action` title. The other titles look to be in a different font/style to our usual titles.

<img width="1410" alt="Screenshot 2021-02-08 at 16 53 38" src="https://user-images.githubusercontent.com/56251247/107253472-40045e00-6a2e-11eb-9645-cfb91cf1eca4.png">

Reply:
Yeah, there are with different styling because I used this -> \`\`
There are props and title in the same time so I thought that using \`\` will show it because quote marks are in use for the code usually

Reply:
@DipperTheDan is right, all the following titles should have ### to be the same size, and they shouldn't be wrapped in `` because they're titles